account_payment_order/migrations/9.0.1.0.0/post-migration.py

```diff
@@ -24,7 +24,9 @@ def map_payment_type(cr):
 @openupgrade.migrate(use_env=True)
 def migrate(env, version):
     cr = env.cr
-    map_payment_type(cr)
+    if openupgrade.table_exists(cr, 'account_payment_order') \
+       and openupgrade.column_exists(cr, 'account_payment_order', 'payment_order_type'):
```

Comment:
Incorrect indentation an line very long (>80 chars)

Reply:
@pedrobaeza Could you please point me to an OCA guideline on how to break line in too long if statement, cause PEP8 is not really strict on that point (see https://www.python.org/dev/peps/pep-0008/#id20 )

Thx

Reply:
Indeed here there's no strict guideline, except complying with that 2 rules (indentation on second line of the if should be double and not > 80 cols). My proposal in this case is:

```python
    table = 'account_payment_order'
    if (openupgrade.table_exists(cr, table) and
            openupgrade.column_exists(cr, table, 'payment_order_type')):
```

Reply:
OK, done